gobblin-runtime/src/main/java/gobblin/runtime/CommandLineLauncher.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2014-2015 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package gobblin.runtime;
+
+import gobblin.runtime.local.LocalJobLauncher;
+import gobblin.util.SchedulerUtils;
+
+import java.util.Properties;
+
+import org.apache.commons.configuration.Configuration;
+import org.apache.commons.configuration.ConfigurationConverter;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Launcher for creating a Gobblin job from command line or IDE.
+ */
+public class CommandLineLauncher {
```

Comment:
Following `CliMRJobLauncher`, can you rename this to `CliLocalJobLauncher` and move to it `gobblin.runtime.local`?